src/shared/model/QueryParams.ts

```diff
@@ -1,6 +1,12 @@
-import { StringifiableRecord } from 'query-string';
 import { ValidClientId } from '../lib/clientId';
 
+export type Stringifiable = string | boolean | number | null | undefined;
```

Comment:
Does `undefined` need to be here? How does it stringify?

```
> undefined.toString();
VM101:1 Uncaught TypeError: Cannot read properties of undefined (reading 'toString')
    at <anonymous>:1:11
```

Reply:
It's more about how the `URLSearchParams` parses these values

```
> new URLSearchParams({foo: undefined, bar: null, baz: false, x: 1, y: "two"})
URLSearchParams {
  'foo' => 'undefined',
  'bar' => 'null',
  'baz' => 'false',
  'x' => '1',
  'y' => 'two' }
  ```

Reply:
It stringifies everything! Hence it's stringifiable!

Reply:
Wow it really does stringify everything.

Reply:
Also if you see how `Stringifiable` is used, without it we wouldn't be able to include any optional parameters in our type definitions.